From 3a9d8de5ecb5e057034d181cf61a1b640ecfdeed Mon Sep 17 00:00:00 2001
From: Kimmo Lehto <klehto@mirantis.com>
Date: Mon, 21 Mar 2022 14:20:23 +0200
Subject: [PATCH] Use go 1.18 (#345)

Upgrade golangci-lint too

SA1019

Unnecessary newline in println

Unused: shlex
---
 .github/workflows/go.yml       | 24 ++++++++++++------------
 .github/workflows/release.yaml |  2 +-
 cmd/flags.go                   |  2 +-
 go.mod                         |  4 ++--
 phase/runhooks.go              |  7 +++++--
 5 files changed, 21 insertions(+), 18 deletions(-)

diff --git a/cmd/flags.go b/cmd/flags.go
index e960d7c..37cf82d 100644
--- a/cmd/flags.go
+++ b/cmd/flags.go
@@ -338,7 +338,7 @@ func fileLoggerHook(logFile io.Writer) *loghook {
 }
 
 func displayLogo(_ *cli.Context) error {
-	fmt.Println(logo)
+	fmt.Print(logo)
 	return nil
 }
 
